packages/cli/src/update.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+import path from 'path';
+import logger from '@percy/logger';
+import { colors } from '@percy/logger/dist/util';
+import pkg from '../package.json';
+
+// filepath where the cache will be read and written to
+const CACHE_FILE = path.join(__dirname, '..', '.releases');
```

Comment:
Any reason to not make this a json file? Would avoid having to stringify and parse it. I was annoyed doing that in the UI & localStorage

Reply:
That would not avoid having to stringify, since `writeFile` still needs a string or buffer.

As for reading and parsing, If you're implying to lean on `require`, that will not work in native ES modules (once we're there). The Node documentation recommends using `fs` like we're doing here, or we could add extra code to enable traditional `require` functionality.

```js
import { createRequire } from "module";
const require = createRequire(import.meta.url);
// ...
let { releases, createdAt } = require('./releases.json');
```

Since it's recommended to explicitly read and parse JSON files anyway, I don't see a reason to add `.json` to the file.